app/controllers/events_controller.rb

```diff
@@ -30,6 +30,18 @@ def create
     end
   end
 
+  def update
+    @event = Event.find(params[:id])
+    authorize @event
+
+    if @event.update(event_params)
+      flash[:notice] = 'Event was successfully updated.'
+      redirect_to @event
+    else
+      render 'edit'
```

Comment:
Totally agree and currently if there are errors in updating/creating events they get rendered via the `app/views/events/_errors.html.erb` from the `app/views/events/_form.html.erb` partial (which in turn is rendered from new & edit views.  That pattern already existed in the project so I carried it forward for the addition of edit/update actions. If you can recommend a better approach for this, I think it would be a good idea to create a new story to update this throughout the project.